.circleci/config.yml

```diff
@@ -12,11 +12,30 @@ aliases:
     command: ~/opennms-container/.circleci/build.sh
   - &registry_login
     name: Login Container Registry
-    command: docker login -u ${CONTAINER_REGISTRY_LOGIN} -p ${CONTAINER_REGISTRY_PASS}
+    command: |
+      docker login -u ${CONTAINER_REGISTRY_LOGIN} -p ${CONTAINER_REGISTRY_PASS}
+      KEY_FOLDER=~/.docker/trust/private
+      mkdir -p $KEY_FOLDER
+      echo "$DELEGATE_PRIVATE_KEY" | base64 -d > $KEY_FOLDER/$DELEGATE_PRIVATE_KEY_NAME.key
+      chmod 600 $KEY_FOLDER/*
+      env | grep DOCKER
+      echo "BASH_ENV: $BASH_ENV"
+      cat $BASH_ENV
+      echo "SHELL: $SHELL"
+      echo $0
+      docker trust key load $KEY_FOLDER/$DELEGATE_PRIVATE_KEY_NAME.key
+  - &bash_env
+    name: Setup Environment Variables
+    command: |
+      echo "export CONTAINER_REGISTRY=docker.io" >> $BASH_ENV
+      echo "export CONTAINER_REGISTRY_REPO=opennmsdcttest" >> $BASH_ENV
```

Comment:
The CONTAINER_REGISTRY (docker.io) and CONTAINER_REGISTRY_REPO/ORGANISATION (opennmsdcttest) are now given by CircleCI config instead of the config in the repository. What I see right now are three options:

* Container registry and organization directly in the repo in a config
* Container registry and organization directly in the CI/CD script
* Container registry and organization as an environment injected by a CircleCI context

Just double checking here if this is the best place to define the target publishing images with just in mind we might want to switch to a different registry structure. It's not super critical, read this here more like an informal comment not as a change request :)

Reply:
If the registry / organization should be settable from the outside then we should rethink how we handle the "no build args in `FROM`" limitation of docker builds with content trust. Some base images have to be addressed by the registry / organization.